lib/PuppeteerSharp/Frame.cs

```diff
@@ -74,10 +76,16 @@ internal Frame(FrameManager frameManager, CDPSession client, Frame parentFrame,
         }
 
         #region Properties
+        internal ICollection<Frame> _childFrames { get; } = new ConcurrentSet<Frame>();
+
         /// <summary>
         /// Gets the child frames of the this frame
         /// </summary>
-        public List<Frame> ChildFrames { get; } = new List<Frame>();
+        public List<Frame> ChildFrames {
+            get {
+                return _childFrames.OrderBy(frame => frame.Id).ToList();
+            }
+        }
```

Comment:
Do we need this?

Reply:
This is an attempt at backwards-compatibility and deterministic order. It is not strictly needed.

Reply:
For example, I think that `FrameManager` expects the order to be deterministic.

Reply:
Specifically `RemoveFramesRecursively` and possibly user code as well.